beacon_node/gossipsub/src/types.rs

```diff
@@ -293,6 +300,13 @@ pub struct Prune {
     pub(crate) backoff: Option<u64>,
 }
 
+/// The node requests us to not forward message ids (peer_id + sequence _number) - IDontWant control message.
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct IDontWant {
+    /// A list of known message ids (peer_id + sequence _number) as a string.
```

Comment:
```suggestion
    /// A list of known message ids.
```
The message-id function is customizable. In Lighthouse its a version of the content hash.

Reply:
updated